studio/components/interfaces/Settings/Vault/Secrets/SecretRow.tsx

```diff
@@ -107,74 +110,71 @@ const SecretRow = ({ secret, onSelectEdit, onSelectRemove }: SecretRowProps) =>
         </p>
-        <Dropdown
-          side="bottom"
-          className="w-[120px]"
-          overlay={
-            <>
-              <Tooltip.Root delayDuration={0}>
-                <Tooltip.Trigger>
-                  <Dropdown.Item
-                    icon={<IconEdit3 size="tiny" />}
-                    disabled={!canManageSecrets}
-                    onClick={() => onSelectEdit(secret)}
-                  >
-                    Edit
-                  </Dropdown.Item>
-                </Tooltip.Trigger>
-                {!canManageSecrets && (
-                  <Tooltip.Portal>
-                    <Tooltip.Content side="bottom">
-                      <Tooltip.Arrow className="radix-tooltip-arrow" />
-                      <div
-                        className={[
-                          'rounded bg-scale-100 py-1 px-2 leading-none shadow',
-                          'border border-scale-200',
-                        ].join(' ')}
-                      >
-                        <span className="text-xs text-scale-1200">
-                          You need additional permissions to edit secrets
-                        </span>
-                      </div>
-                    </Tooltip.Content>
-                  </Tooltip.Portal>
-                )}
-              </Tooltip.Root>
-              <Tooltip.Root delayDuration={0}>
-                <Tooltip.Trigger>
-                  <Dropdown.Item
-                    disabled={!canManageSecrets}
-                    icon={<IconTrash stroke="red" size="tiny" />}
-                    onClick={() => onSelectRemove(secret)}
-                  >
-                    Delete
-                  </Dropdown.Item>
-                </Tooltip.Trigger>
-                {!canManageSecrets && (
-                  <Tooltip.Portal>
-                    <Tooltip.Content side="bottom">
-                      <Tooltip.Arrow className="radix-tooltip-arrow" />
-                      <div
-                        className={[
-                          'rounded bg-scale-100 py-1 px-2 leading-none shadow',
-                          'border border-scale-200',
-                        ].join(' ')}
-                      >
-                        <span className="text-xs text-scale-1200">
-                          You need additional permissions to delete secrets
-                        </span>
-                      </div>
-                    </Tooltip.Content>
-                  </Tooltip.Portal>
-                )}
-              </Tooltip.Root>
-            </>
-          }
-        >
-          <Button asChild type="text" className="px-1" icon={<IconMoreVertical />}>
-            <span></span>
-          </Button>
-        </Dropdown>
+        <DropdownMenu_Shadcn_>
+          <DropdownMenuTrigger_Shadcn_>
+            <Button asChild type="text" className="px-1" icon={<IconMoreVertical />}>
+              <span></span>
+            </Button>
+          </DropdownMenuTrigger_Shadcn_>
```

Comment:
The buttons in this dropdown seem a little short. Maybe can add `asChild` to `DropdownMenuTrigger_Shadcn_` and remove it from the `Button`?

<img width="246" alt="Screenshot 2023-09-15 at 17 16 53" src="https://github.com/supabase/supabase/assets/10985857/a02e4aec-9fd9-42f2-9f7f-6e7e4803542c">

Reply:
Trigger already is asChild i think.

The items look wrong in the dropdown though, they should highlight 100% across

Reply:
we should fix this first actually